src/Microsoft.Health.Fhir.Tests.Common/TestRuntime.cs

```diff
@@ -0,0 +1,106 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.Fhir.Tests.Common
+{
+    public static class TestRuntime
```

Comment:
Perhaps a bigger change, but have wondered if this should be built into our runner somehow? https://github.com/microsoft/fhir-server/blob/main/src/Microsoft.Health.Extensions.Xunit/CustomXunitTestFrameworkExecutor.cs#L177

Reply:
I've added an extension class for IClassFixture.All classes implementing IClassFixture can now have the Retry logic. My only concern in adding it directly to CustomXunitTestFrameworkExecutor, was that probably we would lose control of all requests and maybe ingest more records than required, run methods more time than need, and that could cause some unexpected behaviors.  I think the way how it's right now it's more controllable/manageable. 